server/db/models/Order.js

```diff
@@ -0,0 +1,11 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Order = db.define('order', {
+  grandTotal: {
+    type: Sequelize.FLOAT,
+    allowNull: false
```

Comment:
validate: {notEmpty: true} probably needed?

Reply:
What if the order hasn't been purchased yet and is still in cart?